pkg/ccl/utilccl/jobutils.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Licensed as a CockroachDB Enterprise file under the Cockroach Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
+
+package utilccl
+
+import (
+	"context"
+
+	"github.com/cockroachdb/cockroach/pkg/jobs"
+	"github.com/cockroachdb/cockroach/pkg/sql"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+	"github.com/cockroachdb/cockroach/pkg/sql/sqlbase"
+	"github.com/cockroachdb/cockroach/pkg/sql/types"
+)
+
+// JobExecutionResultHeader is the header for various job commands
```

Comment:
nit: I'm not sure that there is a common _result_ header across all job types, or even if we wanted one, it'd say it is separate from the _queued_ job work, so i might skip this?

Reply:
Sorry, @dt... Just started bors.  I broke out bigger PR where I had detached for all of restore, backup and import.  All of them used the same headers.  That's why I moved it to the utility.

Reply:
Let me try to renamed it.  BulkJobExecutionResultHeader

Reply:
It isn't even common to the bulk jobs though -- import and restore have different heads. I'd skip it entirely.